src/components/Profile/components/EmergencyInfoList/index.scss

```diff
@@ -0,0 +1,46 @@
+.emrg-info-list-header {
+  background-color: #014983;
+  color: #ffffff;
```

Comment:
We generally try to use colors that are defined in `src/vars.scss` so that changing them in that one file will change them across the whole site. See `src/components/Profile/components/PersonalInfoList/index.scss` for an example of using color variables.